ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/util/Env.java

```diff
@@ -37,6 +39,9 @@ public boolean containsKey(String key) {
   }
 
   private Optional<String> optString(String key) {
+    if (!include.contains(key)) {
+      throw new IllegalArgumentException("key missing from include: " + key);
```

Comment:
this should be hit by `Sink*IntegrationTest` if we pull any env vars missing from include, which is a problem I hit again recently `>_<`